### DISCUSSION TITLE
`CONSTRAINT b_c_uk UNIQUE (b, c)` cause a parse error

### DISCUSSION THREAD

**Discussion by KoichiKiyokawa** [sqldef/sqldef#614] [Feature requests]
First, thank you for all of your works in this library.

- MySQL 5.7
- mysqldef v0.13.18
## --export output
```sql
CREATE TABLE `foo` (
  `a` int(11) NOT NULL AUTO_INCREMENT,
  `b` int(11) NOT NULL,
  `c` int(11) NOT NULL,
  PRIMARY KEY (`a`),
  UNIQUE KEY `b_c_uk` (`b`,`c`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

## Input SQL
```sql
CREATE TABLE IF NOT EXISTS `foo` (
  `a` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `b` INT NOT NULL,
  `c` INT NOT NULL,
  CONSTRAINT `b_c_uk` UNIQUE (`b`, `c`)
);
```
## Current output
```
found syntax error when parsing DDL "CREATE TABLE IF NOT EXISTS `foo` (
  `a` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `b` INT NOT NULL,
  `c` INT NOT NULL,
  CONSTRAINT `b_c_uk` UNIQUE (b, c)
)": syntax error at position 151 near 'unique'
```

## Expected output
no error

**Comment by k0kubun** [comment]
Hi :wave: Would you be interested in updating `parser/parser.y` to support this syntax? The steps are:

1. Add a rule to `parser/parser.y`
2. Run `make -C parser` to update `parser/parser.go`

**Comment by KoichiKiyokawa** [comment]
@k0kubun 
I have no knowledge of parser at all 😢,  so I'll pass this task on to someone else.

BTW, I'd like to learn parsers from this article of yours 😁.
https://qiita.com/k0kubun/items/1b641dfd186fe46feb65

**Comment by k0kubun** [comment]
Can you use `UNIQUE KEY b_c_uk (b,c)` instead of `CONSTRAINT b_c_uk UNIQUE (b, c)` in the meantime?

**Comment by gfx** [comment]
Resolved in https://github.com/sqldef/sqldef/pull/1052
